docs/Calendar/BookingsDiagnosticSummary.md

```diff
@@ -0,0 +1,52 @@
+# BookingsDiagnosticSummary
+
+Download the latest release: [BookingsDiagnosticSummary.ps1](https://aka.ms/BookingsDiagnosticSummary)
+
+This script runs a series of tests in a bookings Mailbox (one per execution) and returns a summarized list of the bookings Mailbox characteristics, as well as testing for known configuration issues that can lead to bookings not performing as expected.
```

Comment:
This appears to be geared towards EXO, so we should call that out here that this is only supported for EXO vs OnPrem

Reply:
Added paragraph:
This script only runs on Exchange Online, as Microsoft Bookings is an online only application.

Reply:
making comments unresolved till the commit is pushed.